CHANGELOG.md

```diff
@@ -35,16 +35,32 @@ Swift 5.9
 Swift 5.8
 -----------
 
+* [SE-0378]
+
+  SwiftPM now supports token authentication when interacting with a package registry. The `swift package-registry` command has two new subcommands `login` and `logout` as defined in SE-0378 for adding/removing registry credentials.  
+
+* [#5810]
+
+// TODO
```

Comment:
SwiftPM now allows exposing an executable product that consists solely of a binary target that is backed by an artifact bundle. This allow vending binary executables as their own separate package, independently of the plugins that are using them.